bar.go

```diff
@@ -1,5 +1,5 @@
 package main
 
 func main() {
-	println("hello main")
+	println("hello bar")
 }
```

Comment:
the "start a review" button allows adding multiple review comments, and submitting them only afterwards, apparently.

*edit:* Yep, they can be edited too, and the button "Finish your review (2)" shows the total number of comments with "(Pending)" status.